frontend/src/components/annotation/audio_annotation/audio_annotation_core.vue

```diff
@@ -183,12 +154,14 @@ export default {
                     directory_id: this.$store.state.project.current_directory.directory_id,
                     job_id: this.job_id,
                     attached_to_job: this.task.file.attached_to_job,
+                    task_child_file_id: this.working_file.id,
```

Comment:
Had to add ` task_child_file_id: this.working_file.id` part to make saving work in the context of the task. This logic was copied from `text_annotation_core`